kie-server-parent/kie-server-services/kie-server-services-case-mgmt/src/main/java/org/kie/server/services/casemgmt/CaseManagementServiceBase.java

```diff
@@ -160,6 +160,22 @@ public void cancelCaseInstance(String containerId, String caseId, boolean destro
         }
     }
 
+    public void reopenCase(String caseId, String containerId, String caseDefinitionId, String payload, String marshallingType) {
+        containerId = context.getContainer(containerId, LatestContainerLocator.get()).getContainerId();
```

Comment:
Just a question, shouldn't be here rather ByCaseIdContainerLocator? As case file may contain objects specific to container version.